src/prepare-worksheets/index.html

```diff
@@ -0,0 +1,55 @@
+{% extends "base.html" %}
+{% set active_page = "prepare-worksheets" %}
+
+{% block title %}Owning a Home: Preparation worksheets{% endblock title %}
+
+{% block content %}
+
+<main class="prepare-worksheet content content__1-3" id="main" role="main">
+  <div class="wrap content_wrapper">
+    <aside class="content_sidebar">
+      {% with active_page = "prepare-worksheets" %}
+        {% include "sidebar_nav.html" %}
+      {% endwith %}
+    </aside>
+    <section class="content_main">
+      <h1>Why buy? Why now?</h1>
+      <h2>A worksheet for first-time homebuyers</h2>
+
+      <p class="lead">For many Americans, buying a home can be a great decision. But homeownership isn't for everyone at all times. This worksheet was designed by the CFPB to help you think through your reasons for wanting to buy a home, and to help you evaluate the financial and lifestyle tradeoffs to decide whether now is the right time to buy.</p>
+
+      <section class="type-details">
+
+        <!-- TEMPORARY MOCKED DATA -->
+        <script id="data" type="application/json">
+            [{
+              "text" : "Test text 1",
+              "grade": 0
+            },
+            {
+              "text" : "Test text 2",
+              "grade": 1
+            },
+            {
+              "text" : "Test text 3",
+              "grade": 2
+            }]
+        </script> 
+        
+        <section class="worksheets block block__padded-top block__border-top">
+          <!-- Content filled in dynamically from JS in prepare-worksheet.js -->
+        </section>
+
+        <nav class="block block__padded-top block__border-top">
+          <button class="btn btn-next">
```

Comment:
I recommended this because it seems you're only using this class as a js-hook for your application.  If it's application-specific stuff I think it's best to name it that way. Others who come along looking at the code can more easily identify what pieces are more application-specific.